### ISSUE TITLE
Tornado being imported even if I'm not using tornado in my webserver

### ISSUE THREAD

**Issue by timkpaine** [finos/perspective#2723]
I have a FastAPI-based webserver that I want to ensure my import time and memory footprint is minimized. I see that `tornado` is being imported, even though I'm not using tornado.

### Steps to Reproduce:
```python
import sys
import perspective
"tornado" in sys.modules # True
```

### Expected Result:
```python
import sys
import perspective
"tornado" in sys.modules # False
```

### Actual Result:
```python
import sys
import perspective
"tornado" in sys.modules # True
```

### Environment:
3.0+

**Comment by texodus** [comment]
We can change this, but it's API breaking. If we do, we should also do this for the other handler types and remove the `try`/`except` import logic as it swallows important errors sometimes.

Why is a properly constrained virtual env not the appropriate solution?

**Comment by timkpaine** [comment]
> Why is a properly constrained virtual env not the appropriate solution?

One might very reasonably want tornado in the environment, but not have it imported if not used. I have photoshop on my computer, I'd rather not have it open every time I use slack.

**Comment by texodus** [comment]
Fixed in #2754, released in v3.1.0
